### PR TITLE
commenting ipv6 for sysctls

### PR DESCRIPTION
**Pull request by vijay-vin**
Signed-off-by: Vijay Vinnakota <vijay.vinnakota@ibm.com>

### CODE REVIEW

**Comment by stbenjam** [comment]
/approve
/hold

Release the hold whenever you're ready

**Comment by deepsm007** [comment]
/unhold 
The failing tests are not related to the above PR

File: test/extended/networking/tuning.go [hunk 1]
```diff
@@ -38,8 +38,9 @@ var whitelistedSysctls = []SysctlVariant{
 	{Sysctl: "net.ipv6.conf.IFNAME.accept_redirects", Value: "1", Path: "/proc/sys/net/ipv6/conf/net1/accept_redirects"},
 	{Sysctl: "net.ipv4.conf.IFNAME.secure_redirects", Value: "1", Path: "/proc/sys/net/ipv4/conf/net1/secure_redirects"},
 	// Bumping the net.ipv6 values for Multiarch until https://bugzilla.redhat.com/show_bug.cgi?id=2082239 is fixed in RHEL
-	{Sysctl: "net.ipv6.neigh.IFNAME.base_reachable_time_ms", Value: "30010", Path: "/proc/sys/net/ipv6/neigh/net1/base_reachable_time_ms"},
-	{Sysctl: "net.ipv6.neigh.IFNAME.retrans_time_ms", Value: "1010", Path: "/proc/sys/net/ipv6/neigh/net1/retrans_time_ms"},
+	// uncomment the following two lines once the bug is fixed.
```
Comment (bparees) [inline test/extended/networking/tuning.go:41]: link the bug please

**Pull request merged**